lib/view_component/errors.rb

```diff
@@ -20,12 +20,6 @@ class TemplateError < StandardError
     def initialize(errors, templates = nil)
       message = errors.join("\n")
 
-      if templates
```

Comment:
It's a nit, but we don't need to assign to `message` any more (we didn't before I added the debug output)